vscode-trace-extension/src/trace-viewer-panel/trace-viewer-webview-panel.ts

```diff
@@ -70,6 +70,14 @@ export class TraceViewerPanel {
         return openedPanel;
     }
 
+    public static rerenderPanels() {
+        Object.values(TraceViewerPanel.activePanels).forEach(panel => {
```

Comment:
I think you can call the lines below in method `updateTraceServerUrl()`. With this you can avoid adding this public API.